yt/utilities/command_line.py

```diff
@@ -1017,8 +1017,8 @@ class YTNotebookCmd(YTCommand):
     def __call__(self, args):
         kwargs = {}
         try:
-            # IPython 1.0+
-            from IPython.html.notebookapp import NotebookApp
+            # IPython 4.0+
+            from notebook.notebookapp import NotebookApp
```

Comment:
Let's go ahead and remove the `try/except` and just use the IPython 4.0+ import. I don't think we need to explicitly support IPython 3.x or older, since that means someone hasn't updated IPython since 2015 but is trying to install a yt version from 2018.